test/examples/nodejs_postgresql_test.go

```diff
@@ -0,0 +1,539 @@
+package examples_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/redhat-developer/service-binding-operator/pkg/apis/apps/v1alpha1"
+	"github.com/redhat-developer/service-binding-operator/test/examples/util"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var (
+	exampleName = "nodejs_postgresql"
+	operatorsNS = "openshift-operators"
+	oprName     = "service-binding-operator"
+	appNS       = "service-binding-demo"
+
+	nodeJsApp = "https://github.com/pmacik/nodejs-rest-http-crud"
+	appName   = "nodejs-rest-http-crud"
+
+	clusterAvailable = false
+
+	pkgManifest                      = "db-operators"
+	bckSvc                           = "postgresql-operator"
+	dbName                           = "db-demo"
+	sbr                              = ""
+	podNameSBO, podNameBackingSvcOpr string
+)
+
+func TestNodeJSPostgreSQL(t *testing.T) {
+
+	t.Run("set-example-dir", SetExampleDir)
+	t.Run("get-oc-status", GetOCStatus)
+
+	t.Run("install-service-binding-operator", MakeInstallServiceBindingOperator)
+	t.Run("install-backing-service-operator", MakeInstallBackingServiceOperator)
+	t.Run("create-project", CreateProject)
+	t.Run("import-nodejs-app", ImportNodeJSApp)
+	t.Run("create-backing-db-instance", CreateBackingDbInstance)
+	t.Run("createservice-binding-request", CreateServiceBindingRequest)
+	t.Run("delete-project", DeleteProject)
+	t.Run("uninstall-backing-db-instance", UninstallBackingServiceOperator)
+	t.Run("uninstall-service-binding-operator", UninstallServiceBindingOperator)
+}
+
+//SetExampleDir tests that the current example directory was set as a working directory for running the commands.
+func SetExampleDir(t *testing.T) {
+	examplePath := fmt.Sprintf("%s/%s", util.GetExamplesDir(), exampleName)
+	util.SetDir(examplePath)
+	res := strings.TrimSpace(util.Run("pwd").Stdout())
+	require.Equal(t, examplePath, res)
+}
+
+//Logs the oc status
+func GetOCStatus(t *testing.T) {
+	t.Log("--- Getting OC Status ---")
+	util.GetOutput(util.Run("oc", "status"))
+	clusterAvailable = true
```

Comment:
You have hard-coded `clusterAvailable` as `true`.

Reply:
clusterAvailable flag is used in each of the t.Run function, checking if this flag is set to true to continue the rest of the execution

Reply:
@shvasude but where are you doing the actual check? 
Here, you have just hard-coded the value of `clusterAvailable` to `true`.

Even when `util.GetOutput(util.Run("oc", "status"))` returns that cluster is not available, the control moves to next step and assigns `true` to `clusterAvailable`

Reply:
Agreed, will fix that

Reply:
Added the check by https://github.com/redhat-developer/service-binding-operator/pull/441/commits/ab55675ad467f37fe2d51a1c442e1907a6eb7ac3